packages/nimble-components/src/toggle-button/styles.ts

```diff
@@ -5,87 +5,86 @@ import {
     borderHoverColor,
     borderWidth,
     fillSelectedColor,
+    fillSelectedRgbPartialColor,
     smallDelay
 } from '../theme-provider/design-tokens';
 import { styles as buttonStyles } from '../patterns/button/styles';
 
 export const styles = css`
```

Comment:
The Chromatic diffs for the toggle button matrix story look wrong to me: all of the buttons that were checked and disabled are now rendering as unchecked and disabled. Is that intentional?

![image](https://github.com/ni/nimble/assets/10500124/7a997d74-731d-4a01-85ca-bf363dd559cd)

Reply:
It's intentional, but I need to double check with Brandon that it's correct- Brandon's [toggle button spec] shows the disabled button as being the gray colored background, and it's unclear if this design is for both checked and unchecked toggle buttons. I believe he's back this week, so I'll ask him.

Reply:
I double-checked with Brandon and the prior design was correct- it should have a different appearance when disabled in the checked state. I'm fixing this now 👍

Reply:
Looking better. I noticed that one more state is incorrect: disabled + checked + hovered (unfortunately we don't have automated testing for hover states so had to check in manually). The disabled state should take precedence but I'm still seeing hover feedback for all themes in the "outline" and "ghost" appearance ("block" seems fine). Hover feedback is correctly disabled for regular buttons and anchor buttons.

I was testing in Firefox by hovering over all the "Checked .... Disabled" buttons in the [toggle button theme matrix](https://60e89457a987cf003efc0a5b-gbmhgmvpwd.chromatic.com/?path=/story/tests-toggle-button--toggle-button-theme-matrix)